scripts/mpi_cmake_format

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
On Ubuntu 18.04 the default `python` is still version 2.  I don't like changing this because it can break applications that rely on it (I had problems with this in the past).
Apart from this, it is also easier for the users if the workspace does not need a special virtualenv setup.

Are there cases where using `python3` could cause trouble?

Reply:
with the installation of Vincent I think we switched to python3 by default... Anyway it's not very important I merge this.